### PR TITLE
Add GCP flavour and change AWS flavour to contain also the instance type

### PR DESCRIPTION
**Pull request by tzvatot**
@jhernand @nimrodshn @zgalor @oourfali @igoihman PTAL

### CODE REVIEW

File: model/clusters_mgmt/v1/flavour_type.model [hunk 1]
```diff
@@ -19,13 +19,18 @@ limitations under the License.
 class Flavour {
 	// Default _Amazon Web Services_ settings of the cluster.
 	//
-	// These can be overriden specifying in the clsuter itself a different set of settings.
+	// These can be overridden specifying in the cluster itself a different set of settings.
```
Comment (zgalor) [inline model/clusters_mgmt/v1/flavour_type.model]: not all can be overridden. only the compute instance type

File: model/clusters_mgmt/v1/flavour_type.model [hunk 2]
```diff
@@ -19,13 +19,18 @@ limitations under the License.
...
+	// Default _Google Cloud Platform_ settings of the cluster.
+	//
+	// These can be overridden specifying in the cluster itself a different set of settings.
+	GCP GCPFlavour
+
 	// Number of nodes that will be used by default when creating a cluster that uses
 	// this flavour.
 	//
-	// These can be overriden specifying in the cluster itself a different number of nodes.
+	// These can be overridden specifying in the cluster itself a different number of nodes.
```
Comment (zgalor) [inline model/clusters_mgmt/v1/flavour_type.model]: Again, only the compute nodes can be overridden

Comment (tzvatot) [reply model/clusters_mgmt/v1/flavour_type.model]: Thanks. I'll remove this comment.

File: model/clusters_mgmt/v1/flavour_type.model (hunk 1 above)
Comment (tzvatot) [reply model/clusters_mgmt/v1/flavour_type.model]: Fixed

**Pull request merged**